<?xml version='1.0' encoding='UTF-8'?>
<sdf version="1.6">

  <world name="default">

    <light name="sun" type="directional">

      <cast_shadows>1</cast_shadows>

      <pose frame="">0 0 10 0 -0 0</pose>

      <diffuse>0.8 0.8 0.8 1</diffuse>

      <specular>0.2 0.2 0.2 1</specular>

      <attenuation>

        <range>1000</range>

        <constant>0.9</constant>

        <linear>0.01</linear>

        <quadratic>0.001</quadratic>

      </attenuation>

      <direction>-0.5 0.1 -0.9</direction>

    </light>

    <model name="ground_plane">

      <static>1</static>

      <link name="link">

        <collision name="collision">

          <geometry>

            <plane>

              <normal>0 0 1</normal>

              <size>100 100</size>

            </plane>

          </geometry>

          <surface>

            <friction>

              <ode>

                <mu>100</mu>

                <mu2>50</mu2>

              </ode>

              <torsional>

                <ode/>

              </torsional>

            </friction>

            <contact>

              <ode/>

            </contact>

            <bounce/>

          </surface>

          <max_contacts>10</max_contacts>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <plane>

              <normal>0 0 1</normal>

              <size>100 100</size>

            </plane>

          </geometry>

          <material>

            <script>

              <uri>file://media/materials/scripts/gazebo.material</uri>

              <name>Gazebo/Grey</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

    </model>

    <gravity>0 0 -9.8</gravity>

    <magnetic_field>6e-06 2.3e-05 -4.2e-05</magnetic_field>

    <atmosphere type="adiabatic"/>

    <physics name="default_physics" default="0" type="ode">

      <max_step_size>0.001</max_step_size>

      <real_time_factor>1</real_time_factor>

      <real_time_update_rate>1000</real_time_update_rate>

    </physics>

    <scene>

      <ambient>0.4 0.4 0.4 1</ambient>

      <background>0.7 0.7 0.7 1</background>

      <shadows>1</shadows>

    </scene>

    <spherical_coordinates>

      <surface_model>EARTH_WGS84</surface_model>

      <latitude_deg>0</latitude_deg>

      <longitude_deg>0</longitude_deg>

      <elevation>0</elevation>

      <heading_deg>0</heading_deg>

    </spherical_coordinates>

    <model name="grey_wall">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">0.10925 -2.20748 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_0">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">1.15588 2.12863 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">9.32016 1.76858 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone_clone">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">9.07676 -2.21584 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone_clone_clone">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">0 -7.58257 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone_clone_clone_0">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">2.43635 -5.24448 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone_clone_clone_1">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">1.42516 5.00363 0 0 -0 0</pose>

    </model>

    <model name="grey_wall_clone_clone_clone_1_clone">

      <static>1</static>

      <link name="link">

        <pose frame="">0 0 1.4 0 -0 0</pose>

        <collision name="collision">

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <max_contacts>10</max_contacts>

          <surface>

            <contact>

              <ode/>

            </contact>

            <bounce/>

            <friction>

              <torsional>

                <ode/>

              </torsional>

              <ode/>

            </friction>

          </surface>

        </collision>

        <visual name="visual">

          <cast_shadows>0</cast_shadows>

          <geometry>

            <box>

              <size>7.5 0.2 2.8</size>

            </box>

          </geometry>

          <material>

            <script>

              <uri>model://grey_wall/materials/scripts</uri>

              <uri>model://grey_wall/materials/textures</uri>

              <name>vrc/grey_wall</name>

            </script>

          </material>

        </visual>

        <self_collide>0</self_collide>

        <kinematic>0</kinematic>

      </link>

      <pose frame="">3.92706 6.02039 0 0 0 -1.55775</pose>

    </model>

    <state world_name="default">

      <sim_time>633 198000000</sim_time>

      <real_time>635 50537653</real_time>

      <wall_time>1500987849 920251355</wall_time>

      <iterations>633198</iterations>

      <model name="grey_wall">

        <pose frame="">-4.23621 -3.09696 0 0 -0 0</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">-4.23621 -3.09696 1.4 0 -0 0</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_0">

        <pose frame="">-4.08766 2.3237 0 0 -0 0</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">-4.08766 2.3237 1.4 0 -0 0</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone">

        <pose frame="">7.62251 2.36272 0 0 -0 0</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">7.62251 2.36272 1.4 0 -0 0</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone_clone">

        <pose frame="">7.40322 -2.99971 0 0 -0 0</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">7.40322 -2.99971 1.4 0 -0 0</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone_clone_clone">

        <pose frame="">-0.342361 -6.79895 0 0 0 -1.54779</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">-0.342361 -6.79895 1.4 0 0 -1.54779</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone_clone_clone_0">

        <pose frame="">3.79609 -6.75021 0 0 0 -1.57157</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">3.79609 -6.75021 1.4 0 0 -1.57157</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone_clone_clone_1">

        <pose frame="">-0.557804 6.00447 0 0 0 -1.55775</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">-0.557804 6.00447 1.4 0 0 -1.55775</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="grey_wall_clone_clone_clone_1_clone">

        <pose frame="">3.92706 6.02039 0 0 0 -1.55775</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">3.92706 6.02039 1.4 0 0 -1.55775</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <model name="ground_plane">

        <pose frame="">0 0 0 0 -0 0</pose>

        <scale>1 1 1</scale>

        <link name="link">

          <pose frame="">0 0 0 0 -0 0</pose>

          <velocity>0 0 0 0 -0 0</velocity>

          <acceleration>0 0 0 0 -0 0</acceleration>

          <wrench>0 0 0 0 -0 0</wrench>

        </link>

      </model>

      <light name="sun">

        <pose frame="">0 0 10 0 -0 0</pose>

      </light>

    </state>

    <gui fullscreen="0">

      <camera name="user_camera">

        <pose frame="">14.5969 -11.7438 32.5272 0 1.03964 2.3162</pose>

        <view_controller>orbit</view_controller>

        <projection_type>perspective</projection_type>

      </camera>

    </gui>

  <actor name="actor0"><pose>0 -4 1.02 0 0 0</pose><skin><filename>walk.dae</filename><scale>1</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin filename="/home/kai/catkin_ws/src/Gazebo_ROS_Pedestrians/actor_plugin/lib/libactorplugin_ros.so" name="None"><speed>1.34</speed><target>0 4 1.02</target><dodgingRight>false</dodgingRight><target_weight>1.5</target_weight><obstacle_weight>1.5</obstacle_weight><animation_factor>1.34</animation_factor><ignore_obstacles><model>caffe</model><model>ground_plane</model></ignore_obstacles></plugin></actor><actor name="actor1"><pose>4 0 1.02 0 0 0</pose><skin><filename>stand_up.dae</filename><scale>1</scale></skin><animation name="walking"><filename>walk.dae</filename><scale>1</scale><interpolate_x>true</interpolate_x></animation><plugin filename="/home/kai/catkin_ws/src/Gazebo_ROS_Pedestrians/actor_plugin/lib/libactorplugin_ros.so" name="None"><speed>1.34</speed><target>-4 0 1.02</target><dodgingRight>false</dodgingRight><target_weight>1.5</target_weight><obstacle_weight>1.5</obstacle_weight><animation_factor>1.34</animation_factor><ignore_obstacles><model>caffe</model><model>ground_plane</model></ignore_obstacles></plugin></actor></world>

</sdf>
